docs/en/getting-started/example-datasets/reddit-comments.md

```diff
@@ -0,0 +1,636 @@
+---
+slug: /en/getting-started/example-datasets/reddit-comments
+sidebar_label: Reddit comments
+---
+
+# Reddit comments dataset
+
+This dataset contains publicly-available comments on Reddit that go back to December, 2005, to March, 2023, and contains over 7B rows of data. The raw data is in JSON format in compressed `.zst` files and the rows look like the following:
```

Comment:
This is incorrect.

1. It is not 7B rows of data.
2. It is not available in zst files. zst + bz2 + xz

Reply:
![Screenshot_20230520_022638](https://github.com/ClickHouse/ClickHouse/assets/18581488/3fe4e082-c676-4f23-8c78-838ebaef880d)